grails-app/services/com/unifina/service/StreamService.groovy

```diff
@@ -202,17 +202,17 @@ class StreamService {
 
 	@CompileStatic
 	private boolean isDirectPermissionToStream(SecUser user, Stream stream) {
```

Comment:
This is now too vague. Find usages and replace with direct call to `permissionService.check(user, stream, Operation.STREAM_*)` with `STREAM_*` depending on what context it is.